lib/registry.js

```diff
@@ -89,12 +89,14 @@ function Registry (options) {
   ,   _host = '' + (_options.host || '')    // hostname
   ,   _hive = '' + (_options.hive || HKLM)  // registry hive
   ,   _key  = '' + (_options.key  || '')    // registry key
+  ,   _arch = _options.arch || null         // architecture (32 or 64 bit)
```

Comment:
You should probably default this to `process.arch`, which is 'x64' or 'x86' typically